exasol_integration_test_docker_environment/lib/docker/images/create/utils/file_directory_list_hasher.py

```diff
@@ -129,22 +188,18 @@ def collect_files_and_directories(
                     f"Directory: {directory} contains symlink loops (Symlinks pointing to a parent directory). Please fix!")
             inodes.add(stat.st_ino)
```

Comment:
I think, we should extract the inode symlink stuff into a function

Reply:
done